app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
This error will occurs if host has a null vendor(which is allowed considering the database constraint), but in a model level(when try to do `VENDOR_TYPES[vendor.downcase]` in the method `label_for_vendor`).